tools/src/main/java/org/apache/rocketmq/tools/command/broker/GetBrokerConfigCommand.java

```diff
@@ -83,19 +86,19 @@ public void execute(final CommandLine commandLine, final Options options,
                 Map<String, List<String>> masterAndSlaveMap
                     = CommandUtil.fetchMasterAndSlaveDistinguish(defaultMQAdminExt, clusterName);
 
-                for (String masterAddr : masterAndSlaveMap.keySet()) {
+                for (Entry<String, List<String>> masterAddrEntry : masterAndSlaveMap.entrySet()) {
```

Comment:
masterAddrEntry could be renamed to masterAndSlaveEntry

Reply:
already edited